### PR TITLE
Adding Limbo package to manual data streams

### PR DESCRIPTION
**Pull request by stompsjo**
This PR includes information pertaining to [Limbo](https://limbo-ml.readthedocs.io/en/latest/index.html), a package and dataset for computer vision imagery. Thank you to @mbmcgarry for bringing this to our attention!

(I'll ping @tshead2, who is contributing to Limbo's repository, but feel free to add feedback or ping whoever is maintaining Limbo.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ The process of adding content is detailed in the [contributing page](CONTRIBUTIN
 |[xView](#xview-detection-challenge)              |               |               |               |       x       |               |               |               |               |               |
 |[SpaceNet](#spacenet)                            |               |               |               |       x       |               |               |               |               |               |
 |[COWC](#cars-overhead-with-context)              |               |               |               |       x       |               |               |               |               |               |
+|[Limbo](#limbo)                                  |               |       x       |               |               |               |               |               |               |               |
```
Comment (gonuke) [inline README.md]: did you mean to categorize this as `biota`?

**Review by gonuke — CHANGES_REQUESTED** [review]
I am not sure this dataset has been categorized properly in the table??

File: README.md (hunk 1 above)
Comment (tshead2) [reply README.md]: I'm unfamiliar with this repo, but I'm the maintainer of Limbo and happy to answer questions.

Limbo definitely isn't biota :)  Looking at your categories, I'd go with imaging, and I'd add a new category for depth/LIDAR ... or include those in your definition of hyper spectral.

Cheers,
Tim

Comment (stompsjo) [reply README.md]: Thanks for the feedback! I apparently can't count columns...

This is corrected/included!

**Review by gonuke — APPROVED** [review]

**Pull request merged**